data-transfer/src/main/java/it/eng/datatransfer/model/TransferTerminationMessage.java

```diff
@@ -101,7 +101,7 @@ public TransferTerminationMessage build() {
 	
 	@Override
 	public String getType() {
-		return DSpaceConstants.DSPACE + TransferSuspensionMessage.class.getSimpleName();
+		return DSpaceConstants.DSPACE + TransferTerminationMessage.class.getSimpleName();
```

Comment:
I wonder how these mistakes happened. Nevertheless, nice one for finding and fixing them 👍